.buildkite/commands/test-ios.sh

```diff
@@ -19,36 +21,46 @@ while [ "$INPUT" != "" ]; do
     INPUT="${1-}"
 done
 
-echo '--- :node: Setup Node depenendencies'
+export APP_PATH=./gutenberg/packages/react-native-editor/ios/GutenbergDemo.app.zip
+
+echo "--- 📦 Downloading Build Artifacts"
+download_artifact "GutenbergDemo.app.zip" "./gutenberg/packages/react-native-editor/ios/"
+
+echo '--- :node: Setup Node dependencies'
 npm ci --prefer-offline --no-audit --ignore-scripts
 npm ci --prefix gutenberg --prefer-offline --no-audit
 
 echo '--- :ios: Set env var for iOS E2E testing'
 set -x
 export TEST_RN_PLATFORM=ios
-export TEST_ENV=sauce
+export TEST_ENV=ci
 export JEST_JUNIT_OUTPUT_FILE="reports/test-results/ios-test-results.xml"
 # Set a destination different from the hardcoded one which only works in the
 # older Xcode-setup used by CircleCI
-export RN_EDITOR_E2E_IOS_DESTINATION='platform=iOS Simulator,name=iPhone 13,OS=16.4'
-# This is a relic of the CircleCI setup.
-# It should be removed once the migration to Buildkite is completed.
-export CIRCLE_BRANCH=${BUILDKITE_BRANCH}
+export RN_EDITOR_E2E_IOS_DESTINATION='platform=iOS Simulator,name=iPhone 14,OS=16.4'
```

Comment:
Granted that this is just a test 😄 

But for future reference, it would be useful to interpolate the device here
```suggestion
export RN_EDITOR_E2E_IOS_DESTINATION="platform=iOS Simulator,name=$DEVICE,OS=16.4"
```

Reply:
Thanks for the suggestion! I've updated it a different PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/6336 as for the `RN_EDITOR_E2E_IOS_DESTINATION` variable in `test-ios.sh` it wasn't needed so I've removed it 👍